constraintlayout/compose/src/main/java/androidx/constraintlayout/compose/ToolingUtils.kt

```diff
@@ -97,7 +117,8 @@ private fun addReferencesIds(
     root: ConstraintWidgetContainer,
     rootId: String
 ) {
-    helperWidget.mWidgets.forEach { referencedWidget ->
+    for (i in 0 until helperWidget.mWidgetsCount) {
+        val referencedWidget = helperWidget.mWidgets[i]
```

Comment:
Does this change make any difference?

Reply:
Yes, otherwise it may iterate over non initialized elements in the array.